packages/cli/src/commands/create/index.ts

```diff
@@ -253,30 +251,26 @@ function updatePackageJson(dest: string, projectName: string, flags: CreateOptio
   }
 
   const pkgPath = path.resolve(dest, 'package.json');
-
   const pkgJson = JSON.parse(fs.readFileSync(pkgPath, 'utf8'));
 
   pkgJson.name = projectName;
 
-  updateWorkspaceVersions(pkgJson.dependencies, TUTORIALKIT_VERSION);
-  updateWorkspaceVersions(pkgJson.devDependencies, TUTORIALKIT_VERSION);
```

Comment:
I'm not yet familiar with these CLI commands. This change came from https://github.com/stackblitz/tutorialkit/commit/e444b43035711de4128ce8de79ee3ecb6b5d3294#diff-e85db3b9733aa67073e3397b203523dcd174b6a7b2ee32e2ef6e53933e9b9451.

So the issue here now is that when running `node ./packages/cli/dist/index.js create`, the created tutorial has workspace-protocol in `@tutorialkit` dependencies? It should be using the actual version numbers there?

Reply:
Yes, that's indeed the issue! :smiley: 

Alternatively we could remove the symlink and instead have the build step always copy the template & apply the override. I would be totally fine with that change.

Reply:
I would personally prefer the symlink so we don't copy so many files during dev.

Reply:
Note that in dev, it would only happen when building on the CLI but not when doing `pnpm template:dev`.
 
Right now you don't see the impact of the overrides on dev unless you ran `prepack` instead of `build`. If we remove the symlink and always do the copy it would. 

I think it would also make the CLI a bit simpler.

With all that said I'm fine keeping the symlink.

Reply:
Yea true. I mean as long as dev is not impacted much and we can keep the symlink there then that sounds good to me. Just wanna make sure the DX is still smooth and we avoid file copies if possible.